azurelinuxagent/ga/remoteaccess.py

```diff
@@ -62,7 +62,12 @@ def run(self):
                 if self.incarnation != current_incarnation:
                     # something changed. Handle remote access if any.
                     self.incarnation = current_incarnation
-                    self.remote_access = self.protocol.client.get_remote_access()
+                    try:
+                        self.remote_access = self.protocol.client.get_remote_access()
```

Comment:
This is what we discussed in the meeting - that this code would be reverted. Correct?

Reply:
Nope not this code, that was the RDMA code. In this I added a log.warn incase there was a ProtocolError but reverted that change because this file in majorly empty